lib/ripper-tags.rb

```diff
@@ -122,6 +124,9 @@ def self.process_args(argv, run = method(:run))
       elsif !options.recursive then abort(optparse.banner)
       end
       options.format ||= File.basename(options.tag_file_name) == "TAGS" ? "emacs" : "vim"
+      if options.format == 'emacs' and not options.provide_tag_file_name
```

Comment:
I guess the Ruby code for this could be:

```rb
options.tag_file_name ||= options.format == 'emacs' ? "TAGS" : "./tags"
```

And this existing line should be moved below, because it's going to fail with `options.tag_file_name` being nil by default:

```rb
options.format ||= File.basename(options.tag_file_name) == "TAGS" ? "emacs" : "vim"
```

Thanks!